core/src/test/java/org/apache/lucene/queries/BlendedTermQueryTests.java

```diff
@@ -205,7 +206,7 @@ public void testBasics() {
     public Term[] toTerms(String[] fields, String term) {
         Term[] terms = new Term[fields.length];
         List<String> fieldsList = Arrays.asList(fields);
-        Collections.shuffle(fieldsList, random());
+        Randomness.shuffle(fieldsList);
```

Comment:
And if this is because you want the tests.seed check, please, add such a check somewhere else (such as base test class) and instead keep all tests simple.